x-pack/plugins/stack_connectors/common/slack_api/lib.ts

```diff
@@ -77,3 +77,15 @@ export function retryResultSeconds(
     serviceMessage: message,
   };
 }
+
+// This is only modifiable via config, at startup.  The reason we
```

Comment:
This comment is actually wrong - I originally thought we could create a `kibana.yml` config setting to override for testing, but that got a little complicated.  We should change this comment to indicate this API is only used for testing.